CRVResponse/fcl/prolog_v08.fcl

```diff
@@ -42,7 +42,7 @@ BEGIN_PROLOG
       scintillationYieldVariationCutoff : 0.4   //the scintillation yield variation is cut off at 40% below the mean
       startTime                         : 400.0    //0.0 ns
       visibleEnergyAdjustmentFileName   : "CRVResponse/fcl/visibleEnergyAdjustment.txt"
-      timeOffsets                       : { inputs : [ @sequence::CommonMC.TimeMaps ] }
+      timeOffsets                       : { inputs : [ @sequence::TIME_MAPS ] }
```

Comment:
Why are you overriding CommonMC with a local variable?  That makes the behavior non-standard.   This looks like a SU2020-specific hack.

Reply:
The time maps have been reverted here.